constraint_utils/generic.py

```diff
@@ -160,6 +168,14 @@ def rebin_methylation(t: Union[hl.MatrixTable, hl.Table], bins: int=20) -> Union
 
 
 def trimer_from_heptamer(t: Union[hl.MatrixTable, hl.Table]) -> Union[hl.MatrixTable, hl.Table]:
+    """Timming the context column in input table to 3 bases.
```

Comment:
a more accurate description would be "Trim heptamer context to create trimer context" or something similar, and then note that this function adds a `context` annotation that is 3 bases